base/base.ts

```diff
@@ -49,10 +49,13 @@ export interface FrontendCBK {
       params?: Record<string, unknown>
     ) => Promise<T>;
   };
-  getElementValue(ref: string): string;
+  // This one needs to be any because the general return is string however
```

Comment:
why don't `string | string[]` then?

Reply:
Hmm, the return from ListInput currently is as so [ { <option_ref>: <option_value> } ] where the ref is actually dynamic. So hence why changing it to any :') 

Thoughts? @ivanyeoh

Reply:
How about like this @ianneo97 :
`string | Record<string, string>[]`